app/src/main/java/mollie/tictactoe/MenuActivity.java

```diff
@@ -31,11 +36,48 @@ private void setBoardOptionButton() {
         mBoardSizeButton.setOnClickListener(view -> mGameOptions.putExtra(EXTRA_BOARD_TYPE, 3));
     }
 
-    private void setGameOptionButton() {
-        mGameOptionButton.setOnClickListener(view -> mGameOptions.putExtra(EXTRA_PLAYER_TYPES, new String[]{"HUMAN", "HUMAN"}));
-    }
-
     private void setPlayButton() {
         mPlayButton.setOnClickListener(view -> startActivity(mGameOptions));
     }
+
+    public void onRadioButtonClicked(View view) {
```

Comment:
This looks like quite a long method. Can it be extracted into it's own class, passing in the view.id, the R.id and returning the state to set in a hash or small data class? This might help distance Android classes from Mollie classes also.